src/algebra/quotient.lean

```diff
@@ -0,0 +1,53 @@
+/-
+Copyright (c) 2021 Anne Baanen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anne Baanen
+-/
+
+import data.set_like.basic
+
+/-!
+# Algebraic quotients
+
+This file defines notation for algebraic quotients, e.g. quotient groups `G /// H`,
+quotient modules `M /// N` and ideal quotients `R /// I`.
+
+The actual quotient structures are defined in the following files:
+ * quotient group: `src/group_theory/quotient_group.lean`
+ * quotient module: `src/linear_algebra/quotient.lean`
+ * quotient ring: `src/ring_theory/ideal/quotient.lean`
+
+## Notations
+
+The following notation is introduced:
+
+* `G /// H` stands for the quotient of the type `G` by the `set_like` element H.
```

Comment:
It doesn't actually require `set_like` though, right? Perhaps we could have `T /// s` be notation for `quotient s` when `s : setoid T` too?

Reply:
Indeed, that was a remnant of an older design. `has_quotient` doesn't put any real restrictions on `H`'s type, so here's my attempt at not being too vague:
```suggestion
* `G /// H` stands for the quotient of the type `G` by some term `H` 
  (for example, `H` can be a normal subgroup of `G`).
```